runtime/opt/zalando/runtime/Docker

```diff
@@ -29,6 +30,15 @@ def get_env_options(config: dict):
         yield '-e'
         yield 'ETCD_URL=http://172.17.42.1:2379'
 
+    yield '-e'
+    yield 'APPLICATION_ID={}'.format(config.get('application_id'))
+
+    yield '-e'
+    yield 'APPLICATION_VERSION={}'.format(config.get('application_version'))
+
+    yield '-e'
+    yield 'INSTANCE_ID={}'.format(boto.utils.get_instance_metadata()['instance-id'])
```

Comment:
I'm using these information in https://github.com/zalando-stups/friboo/pull/15/ to build unique log file names.

I think it's useful for many applications to know about their environment. "INSTANCE_ID" could also have been called "HOST_ID" or whatever. If we do not pass the information to the docker process, but use the EC2 metadata service from inside the application, we would bind all of our apps to AWS. This would IMHO make the vendor locking even worse.